cocos/animation/core/pose.ts

```diff
@@ -0,0 +1,203 @@
+import { lerp } from '../../core';
+import { assertIsTrue } from '../../core/data/utils/asserts';
+import { Transform, __applyDeltaTransform, __calculateDeltaTransform } from './transform';
+import { TransformArray } from './transform-array';
+
+export class Pose {
+    readonly transforms: TransformArray;
+
+    readonly metaValues: Float64Array;
+
+    protected constructor (transforms: TransformArray, metaValues: Float64Array) {
+        this.transforms = transforms;
+        this.metaValues = metaValues;
+    }
+
+    /**
+     * @internal
+     */
+    public static __create (transforms: TransformArray, metaValues: Float64Array) {
+        return new Pose(transforms, metaValues);
+    }
+}
+
+export class TransformFilter {
+    constructor (involvedTransforms: readonly number[]) {
+        this._involvedTransforms = new Uint32Array(involvedTransforms);
```

Comment:
This value is a typedarray of transform index, UInt32 seems to be too large, Uint16 is enough?

Reply:
Adjusted here: https://github.com/cocos/cocos-engine/pull/13503/commits/ad6a109de0e596abe1091f099c91f68e4dc00950